development/tools/cmd/gcscleaner/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/kyma-project/test-infra/development/tools/pkg/gcscleaner"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	ctx := context.Background()
+
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	cfg, err := readConfig()
+	if err != nil {
+		logrus.Fatal(errors.Wrap(err, "reading configuration"))
+	}
+
+	err = gcscleaner.Clean(ctx, cfg, stiface.AdaptClient(client))
+	if err != nil {
+		logrus.Fatal(errors.Wrap(err, "cleaning buckets"))
+	}
+}
+
+var (
+	argProjectName                string
+	argExcludedBucketNames        string
+	argBucketLifespanDuration     string
+	bucketLifespanDurationDefault = "2h"
+
+	// ErrInvalidProjectName - returned if project name argument is invalid
+	ErrInvalidProjectName = errors.New("invalid project name argument")
+	// ErrInvalidDuration - returned if duration argument is invalid
+	ErrInvalidDuration = errors.New("invalid duration argument")
+)
+
+func init() {
+	flag.StringVar(
```

Comment:
What would be a benefit of using `envconfig` instead of standard library in this particular case?

Reply:
- Cleaner configuration structure
- You don't need to load config manually

Reply:
In my opinion 
- the standard library is just fine in the scope of the application
- we still need to have tests that will verify that args were passed properly (I'm not talking about testing `envconfig`)
- the standard library is used widely across other tools in `test-infra` project

Reply:
The `envconfig` is also widely used in Kyma projects. It would reduce lines from 69 to 155, so it's quite a lot of unnecessary code.

> we still need to have tests that will verify that args were passed properly (I'm not talking about testing envconfig

Not if we used `envconfig`, as we don't test external dependencies.